src/WebJobs.Script/Config/WorkerConcurrencyOptionsSetup.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Azure.WebJobs.Script.Workers.Rpc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    internal class WorkerConcurrencyOptionsSetup : IConfigureOptions<WorkerConcurrencyOptions>
+    {
+        private readonly IConfiguration _configuration;
+        private readonly IEnvironment _environment;
+
+        public WorkerConcurrencyOptionsSetup(IConfiguration configuration, IEnvironment environment)
+        {
+            _configuration = configuration;
+            _environment = environment;
+        }
+
+        public void Configure(WorkerConcurrencyOptions options)
+        {
+            if (string.IsNullOrEmpty(_environment.GetEnvironmentVariable(RpcWorkerConstants.FunctionsWorkerProcessCountSettingName))
+               && string.IsNullOrEmpty(_environment.GetEnvironmentVariable(RpcWorkerConstants.PythonThreadpoolThreadCount))
```

Comment:
Does this mean that the presence of either PythonThreadpoolThreadCount or PSWorkerInProcConcurrencyUpperBound app settings effectively turns the feature off? I think we should be able to preserve the feature while respecting these settings. I'm ok with merging this code as is, but let's take a note to discuss/fix this later.

Not sure about FunctionsWorkerProcessCountSettingName - this one probably should turn the feature off.

Reply:
Added 2 flags:
```
RemoveConcurrencyLimits 
DynamicConcurrencyEnabled
```